src/components/SearchBar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "@emotion/styled";
+import FormInput from "./FormInput";
+import LocationIcon from "../assets/images/icons/icon-location.svg";
+import SearchIcon from "../assets/images/icons/icon-search.svg";
+
+const Row = styled.div`
+  display: flex;
+  padding: ${(props) => props.theme.layout.space400};
+`;
+
+const Column = styled.div`
+  flex: ${(props) => props.flex || 1};
+  padding: ${(props) => props.theme.layout.space100};
+  border: 1px solid #ccc;
```

Comment:
Can you please add this new color `#cccccc` to the `theme/light.js` file? Please use the same method as the other color definition so we can support dark mode.